src/app/shared/request/request.service.ts

```diff
@@ -98,16 +98,16 @@ export class RequestService {
   private getEndpointUrl(endpoint, isLoginAPI?: boolean) {
     let endpointUrl = '';
     if (isLoginAPI) {
-      endpointUrl = this.loginApiUrl + endpoint;
+      endpointUrl = this.utils.urlFormatter(this.loginApiUrl, endpoint);
     } else if (this.storage.stackConfig && this.storage.stackConfig.coreApi) {
-      endpointUrl = this.storage.stackConfig.coreApi + endpoint;
+      endpointUrl = this.utils.urlFormatter(this.storage.stackConfig.coreApi, endpoint);
     } else {
       throw new Error('Can not find API URL.');
     }
     if (endpoint.includes('https://') || endpoint.includes('http://')) {
```

Comment:
since we have the urlFormatter, we don't need to check the http here anymore

Reply:
@shawnm0705 
in some places we pass full url as endpoint ( "http://abcd.com/123" ). like the URL we use to get user location ("https://ipapi.co/json"). this is for check that. to not add our API URL in to those urls.
if endpoint have http or https then not adding any url to endpoint.

Reply:
inside the urlFormatter, we also checks if it has `http`, so this is redundant I think?

Reply:
I can't find any place we use this like this. let's remove this as you mention, if there any issue came we can fix this later